Makefile

```diff
@@ -66,7 +66,11 @@ lint-go:
 
 .PHONY: lint-proto
 lint-proto:
+	# Vendor go dependencies to build protobuf definitions
+	go mod vendor
```

Comment:
We should probably add `vendor/` to the `.gitignore` file if we're going to do this.

Reply:
`.gitignore` already has `vendor/`: https://github.com/akuity/kargo/blob/ebf0dfc8224ba2414999da14f8ffdaf95c2eee3c/.gitignore#L7

Reply:
I looked. idk how I missed it. So sorry!